src/functionHandlers/notarisePdt/v2/handler.ts

```diff
@@ -90,14 +85,20 @@ export const main: Handler = async (
           personalData === null;
 
         personalData?.vaultData.forEach((vault) => {
-          if (vault.dateofbirth === dob && vault.gender === gender) {
+          if (
+            parseDateWithoutZeroes(vault.dateofbirth) === dob &&
+            vault.gender === gender
+          ) {
             isDobAndGenderInVault = true;
             traceWithRef(`Dob and gender match with vault data`);
           }
         });
 
         personalData?.manualData.forEach((manual) => {
-          if (manual.dateofbirth === dob && manual.gender === gender) {
+          if (
+            parseDateWithoutZeroes(manual.dateofbirth) === dob &&
```

Comment:
whats this all extra spaces...

Reply:
it's based on the prettierrc `printWidth`, do we want to change it?

Reply:
oh okay if it pass lint then is fine